### ISSUE TITLE
Additional map controls

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#116]
## User story
1. As a User
2. I want additional map controls
3. So that I can select regions / polygons 
4. And I can have a search and settings button

## Acceptance criteria
* Select polygon 
* Get data about a selected region
* Ability to click icons without changing map location
* How the additional control is displayed is not exacly specified yet

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by op-hub** [comment]
To big of a feature: split into smaller chunks, 
Decision: close this duplicate
